js/background.js

```diff
@@ -3,7 +3,8 @@
  *
  * This only does anything when the bookmarking button is clicked
  */
-chromeOrBrowser().browserAction.onClicked.addListener(function(tab) {
+var executedScript = false;
```

Comment:
This is here to track if we've already called `executeScript`. If it's not been called, call it and then in the callback, set this to true.